docs/book/storage/adapter.md

```diff
@@ -561,6 +561,50 @@ Name | Data Type | Default Value | Description
 ---- | --------- | ------------- | -----------
 `namespace_separator` | `string` |  ":" | A separator for the namespace and prefix.
 
+## The BlackHole Adapter
+
+`Laminas\Cache\Storage\Adapter\BlackHole` **does not** store any cache items.
+
+This adapter implements the following interfaces:
+
+- `Laminas\Cache\Storage\StorageInterface`
+- `Laminas\Cache\Storage\AvailableSpaceCapableInterface`
+- `Laminas\Cache\Storage\ClearByNamespaceInterface`
+- `Laminas\Cache\Storage\ClearByPrefixInterface`
+- `Laminas\Cache\Storage\ClearExpiredInterface`
+- `Laminas\Cache\Storage\FlushableInterface`
+- `Laminas\Cache\Storage\IterableInterface`
+- `Laminas\Cache\Storage\OptimizableInterface`
+- `Laminas\Cache\Storage\TaggableInterface`
+- `Laminas\Cache\Storage\TotalSpaceCapableInterface`
+
+### Capabilities
+
+Capability | Value
+---------- | -----
+`supportedDatatypes` | `null`, `bool`, `int`, `float`, `string`, `array`, `object`
+`supportedMetadata` | none
+`minTtl` | 0 or 1, depending on `psr` option.
+`maxTtl` | 0
+`staticTtl` | `false` or `true`, depending on `psr` option
+`ttlPrecision` | 1
+`useRequestTime` | false
+`lockOnExpire` | 0
+`maxKeyLength` | -1
+`namespaceIsPrefix` | `true`
+`namespaceSeparator` | none
+
+### Adapter specific Options
```

Comment:
```suggestion
### Adapter Specific Options
```

Reply:
Should I update all other adapter specific options aswell?

Reply:
I wanted to spare you that pain because this also means to remove "The" from all headlines. 😉

Reply:
Fixed ;-)